pytorch_lightning/utilities/cloud_io.py

```diff
@@ -58,5 +58,10 @@ def atomic_save(checkpoint, filepath: str):
         torch.save(checkpoint, bytesbuffer, _use_new_zipfile_serialization=False)
     else:
         torch.save(checkpoint, bytesbuffer)
+
+    assert checkpoint
+    assert bytesbuffer.getvalue()
```

Comment:
this second assertion fails. 
how can it be that the checkpoint here does not get written to the bytes buffer?